src/sage/groups/abelian_gps/abelian_group.py

```diff
@@ -1708,16 +1715,19 @@ def __init__(self, ambient, gens, names="f", category=None):
         H = libgap(ambient).Subgroup(H_gens)
 
         invs = H.TorsionSubgroup().AbelianInvariants().sage()
+        gens_orders = tuple([ZZ(order_sage) for g in H.GeneratorsOfGroup()
+                    if (order_sage := g.Order().sage()) is not infinity])
```

Comment:
```suggestion
        gens_orders = tuple([ZZ(order_sage) for g in H.GeneratorsOfGroup()
                             if (order_sage := g.Order().sage()) is not infinity])
```
Better alignment IMO (and we generally do this (now) in the Sage source from PEP8).